### DISCUSSION TITLE
`Sparrow.LowMemory.LowMemoryNotification` messages in the logs

### DISCUSSION THREAD

**Discussion by n-bilalis** [ravendb/ravendb#17623] [Q&A] [answered]
I have a 5.4.110 server, running on Docker, and on the `Manage Server / Admin Logs` screen I get too many entries like the following:

> 2023-11-01T07:41:16.9897818Z, 15, Operations, Server, Sparrow.LowMemory.LowMemoryNotification, Running 102 low memory handlers with severity: None. Commit charge: 4.975 GBytes / 5.773 GBytes, Memory: 2.893 GBytes / 5.773 GBytes, Available memory for processing: 4.052 GBytes, Dirty memory: 0 Bytes, Managed memory: 45.81 MBytes, Unmanaged allocations: 4.43 MBytes, Lucene managed: 14.62 KBytes, Lucene unmanaged: 84 KBytes

Mem usage seems normal:

![image](https://github.com/ravendb/ravendb/assets/113581417/4207331c-5e40-4464-87fb-82fb7dadd278)

I saw [another post](https://github.com/ravendb/ravendb/discussions/17448) that mentioned this problem, and in the answer they said that this will be fixed in the next stable version of `6.0`. Is this something that is coming `5.4` too?

**Comment by garayx** [comment]
Hi 

This is just a log about Running 102 low memory handlers
The actual amount of free memory which is left for the RavenDB Server, is also displayed in this log (see: `Available memory for processing: 4.052 GBytes`): 

![image](https://github.com/ravendb/ravendb/assets/16625387/b8761971-193a-477c-acb2-e2048c6f9000)

The post you mentioned, talks regarding failing to [import dump file](https://ravendb.net/docs/article-page/6.0/csharp/studio/database/tasks/import-data/import-data-file) and has an actual `EarlyOutOfMemoryException` thrown, do you have this issue as well ?

**Reply by n-bilalis** [reply]
No, I have no exceptions happening.

I was under the impression that the post I mention had two unrelated issues, since the solution states `Fixes for both points`, but I am probably mistaken.

So, are those entries something to worry about?

**Comment by grisha-kotler** [comment]
The memory severity in the log is `None`. Low memory states are `Low` or `ExtremelyLow`.

**Reply by n-bilalis** [reply]
I haven't noticed the `severity: None` part, to be honest.

So, is it normal to get those `LowMemoryNotification`s every few seconds, even when there is no actual problem with low memory?

**Reply by grisha-kotler** [reply, marked as answer]
Yes. When you open the admin logs in the Raven Studio, it will turn on the information logs.
By default, the operations logs are turned on and you'll see this message (in the log files) only when you are in a low memory state.
https://ravendb.net/docs/article-page/5.4/csharp/server/troubleshooting/logging
